src/D2L.CodeStyle.Analyzers/Common/MutabilityInspector.cs

```diff
@@ -83,6 +83,11 @@ public sealed class MutabilityInspector {
 			MutabilityInspectionFlags flags,
 			HashSet<ITypeSymbol> typeStack
 		) {
+			if( type is IErrorTypeSymbol  || type == null ) {
+				// type can't be resolved; the project is either not configured correctly, or there's a typo
+				return false;

```

Comment:
Should we be marking types we don't know about as immutable? That seems hazardous